SteakBot.Core/Services/MemeService.cs

```diff
@@ -52,6 +66,37 @@ public bool EditCommand(string oldCommandName, MemeCommand newCommand)
             return true;
         }
 
+        public MemeResultType GetMemeType(string value)
+        {
+            var valueToProcess = value.ToLower();
+
+            // totally original regex (https://stackoverflow.com/questions/3809401/what-is-a-good-regular-expression-to-match-a-url)
+            Regex urlRegex = new Regex(@"(http(s)?:\/\/.)?(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&\/\/=]*)");
+            if(urlRegex.IsMatch(valueToProcess))
+            {
+                var isImage = _imageFormats.Any(imageFormat => valueToProcess.EndsWith(imageFormat));
+                var isVideo = _videoFormats.Any(videoFormat => valueToProcess.EndsWith(videoFormat));
+
+                if (isImage && isVideo)
+                {
+                    return MemeResultType.Unknown;
+                }
+
+                if (isImage)
+                {
+                    return MemeResultType.Image;
+                }
+                else
+                {
+                    return MemeResultType.Video;
```

Comment:
When the value is a link but both `isImage` and `isVideo` are `false` we end up here. Is this intentional? Doesn't look right to me.

Reply:
Corrected.